File: Orien.PYS.Web/Orien.PYS.Data/Entity/Group.cs
﻿namespace Orien.PYS.Data.Entity
{
    public class Group
    {
        /// <summary>
        /// Gets or Sets Id.
        /// </summary>
        public long Id { get; set; }

        /// <summary>
        /// Gets or Sets Name.
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        /// Gets or Sets Unique Identity.
        /// </summary>
        public string UId { get; set; }

        /// <summary>
        /// Gets or Sets Description.
        /// </summary>
        public string Description { get; set; }

        /// <summary>
        /// Gets or Sets Admin.
        /// </summary>
        public string Admin { get; set; }

        /// <summary>
        /// Gets or Sets Created On.
        /// </summary>
        public DateTime Created_on { get; set; }

        /// <summary>
        /// Gets or Sets Updated on.
        /// </summary>
        public DateTime Updated_on { get; set; }
    }
}

File: Frontend/src/app/shared/Models/Group.ts
import { Users } from "./Users"

export interface GroupResponse{
    err: boolean;
    errMessage: string;
    data: Group[]
}

export interface GroupResponseOne{
    err: boolean;
    errMessage: string;
    data: Group
}

export interface GroupAddResponseOne{
    err: boolean;
    errMessage: string;
    data: string
}

export interface Group{
    Group_id: number
    Name: string
    Description: string
    Admin: Users
    Members: Users[]
    Created_on: string
    Updated_on: string
}